_projects/midas.md

```diff
@@ -4,9 +4,21 @@ collection: projects
 permalink: /projects/midas
 abstract: "Supporting new programming and execution models for data-intensive analysis in a wide range of science and engineering applications."
 status: active
+logo: midas_logo.png
 ---
 
-Middleware for Data-Intensive Analytics and Science (MIDAS) supports new programming and execution models for data-intensive analysis in a wide range of science and engineering applications. MIDAS provides a scalable runtime system and appropriate resource management abstractions enabling [SPIDAL](http://www.spidal.org) libraries. In addition, MIDAS provides the underlying resource management middleware and heterogeneous infrastructure access layer to support streaming-based applications on High Performance Computing resources.
-
-<a href="https://github.com/radical-cybertools/midas"><i class="fa fa-github"></i>GitHub Repository</a><br>
-<a href="https://www.nsf.gov/awardsearch/showAward?AWD_ID=1443054"><i class="fa fa-nsf"></i>NSF Award</a>
+<div class="project-description">
+	<figure>
+		<img src="../images/projects/midas.png" width="500">
```

Comment:
Here the user will not need to add the exact name. The page can pick it up from the header
```suggestion
		<img src="{{site.baseurl}}/images/projects/{{page.logo}}" width="250">
```